helix-term/src/ui/markdown.rs

```diff
@@ -6,48 +6,46 @@ use tui::{
 
 use std::sync::Arc;
 
-use pulldown_cmark::{CodeBlockKind, CowStr, Event, Options, Parser, Tag};
+use pulldown_cmark::{CodeBlockKind, CowStr, Event, HeadingLevel, Options, Parser, Tag};
 
 use helix_core::{
     syntax::{self, HighlightEvent, Syntax},
     Rope,
 };
 use helix_view::{
-    graphics::{Margin, Rect},
+    graphics::{Margin, Rect, Style},
     Theme,
 };
 
 pub struct Markdown {
     contents: String,
 
     config_loader: Arc<syntax::Loader>,
-
-    text_style: String,
-    block_style: String,
-    heading_style: String,
 }
 
 // TODO: pre-render and self reference via Pin
 // better yet, just use Tendril + subtendril for references
 
 impl Markdown {
+    // theme keys, including fallbacks
+    const TEXT_STYLE: [&'static str; 2] = ["markup.normal", "markup"];
```

Comment:
I got rid of this on `master` and only use `ui.text` since `markup.normal` isn't a scope we use.

Reply:
Thanks, I rebased on top of `master` and changed things accordingly.